### PR TITLE
Created application to store bets, added User model and his dao

### PR DESCRIPTION
**Pull request by vicire**

### CODE REVIEW

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int betValue = Integer.parseInt(betData[0]);
+                double coefficient = Double.parseDouble(betData[1]);
+                bet = new Bet(betValue, coefficient);
+            } catch (NumberFormatException e) {
+                System.out.println("Введіть будь ласка коректні дані для розрахунку ставки");
+            }
+            betDao.add(bet);
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: even if input is incorrect, you still save this data in db

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: how do you save users into db?

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Nope, coz in try block I will get an exception, so my `bet` still null and I add to the storage null.

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: in my BetDaoImpl I have method add, which add the bet to the storage

Comment (Max-Pochepets) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: yes, BUT: you also have User.class. where is it used?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (Max-Pochepets) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: > Nope, coz in try block I will get an exception, so my `bet` still null and I add to the storage null.

and why do u need null value in the storage?

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: There no requirements in task to do this, but ok, I will add it)

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/BetConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class BetConsoleHandler {
+    private static final String EXIT = "q";
+    private static final String DELIMITER = " ";
+    private static final int BET_VALUE_INDEX = 0;
+    private static final int COEFFICIENT_INDEX = 1;
+    BetDao betDao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/BetConsoleHandler.java]: access modifier

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class UserConsoleHandler {
+    private static final String EXIT = "q";
+    private static final String DELIMITER = " ";
+    private static final int NAME_VALUE_INDEX = 0;
+    private static final int AGE_INDEX = 1;
+    UserDao userDao = new UserDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: same

File: src/main/java/core/basesyntax/controller/BetConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введіть розмір ставки і її коефіціент або 'q' для виходу");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/BetConsoleHandler.java]: let's write in english

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase(EXIT)) {
+                return;
+            }
+            try {
+                String[] userData = command.split(DELIMITER);
+                String name = userData[NAME_VALUE_INDEX];
+                int age = Integer.parseInt(userData[AGE_INDEX]);
+                userDao.add(new User(name,age));
+                System.out.println(userDao.getAll());
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: What is printed in this case?

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java (hunk 6 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: all info from user storage. Do we have to print only the nearest user info?

File: src/main/java/core/basesyntax/controller/BetConsoleHandler.java (hunk 3 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/BetConsoleHandler.java]: Fixed

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java (hunk 4 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: Fixed

File: src/main/java/core/basesyntax/controller/BetConsoleHandler.java (hunk 5 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/BetConsoleHandler.java]: Fixed

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java (hunk 6 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: that's okay, but let's use forEach() with sout to print each user in its own line

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/UserConsoleHandler.java (hunk 6 above)
Comment (vicire) [reply src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: ok)

Comment (vicire) [reply src/main/java/core/basesyntax/controller/UserConsoleHandler.java]: Fixed

**Pull request merged**